### ISSUE TITLE
It seems that there is no pre-trained model and dataset

### ISSUE THREAD

**Issue by MorenMoren** [sx-zhang/T-diff#2]
I sincerely appreciate your outstanding work! 
However, there is no folders containing the pre-trained model and the dataset. 
will you publish them?

**Comment by sx-zhang** [comment]
Thank you for your kind words! The pre-trained model has been updated, and you can now test its performance on the validation set.
